### PR TITLE
ft: ZENKO-1343 update ingestion owner md

### PR DESCRIPTION
**Pull request by philipyoo**

### CODE REVIEW

**Review by JianqinWang — APPROVED** [review]

**Review by dora-korpar — APPROVED** [review]

File: extensions/mongoProcessor/MongoQueueProcessor.js [hunk 1]
```diff
@@ -109,6 +109,80 @@ class MongoQueueProcessor {
...
+     * @param {function} done - callback(error)
+     * @return {undefined}
+     */
+    _processObjectQueueEntry(sourceEntry, ownerInfo, done) {
+        const bucket = sourceEntry.getBucket();
+        // always use versioned key so putting full version state to mongo
+        const key = sourceEntry.getObjectVersionedKey();
+
+        // update necessary metadata fields before saving to Zenko MongoDB
+        this._updateOwnerMD(sourceEntry);
```
Comment (rahulreddy) [inline extensions/mongoProcessor/MongoQueueProcessor.js]: I think you forgot to pass `ownerInfo` here.

Comment (rahulreddy) [reply extensions/mongoProcessor/MongoQueueProcessor.js]: Added a commit.

**Review by rahulreddy — APPROVED** [review]

**Comment by rahulreddy** [comment]
@bert-e help

**Comment by rahulreddy** [comment]
@bert-e bypass_author_approval

**Pull request merged**